### ISSUE TITLE
More clear, up-front consequences will likely be needed

### ISSUE THREAD

**Issue by astearns** [w3c/TPAC2022-Health-rules#10]
https://www.w3.org/2022/09/TPAC/health.html has a number of suggestions and rules for participation. As far as I can tell, in-person mask-wearing is the only rule with a clearly stated consequence. We probably will need more.

TPAC has this rule:

> Wear a mask during your entire travel journey (onboard a flight, train or federally-regulated vessel, as well as in airports and other points of entry)

The CSSWG had a similar rule for our meeting last week:

> Wear an N95 mask during travel. Keep it on 100% in the airport; have your drinks and snacks in-flight on the airplane only.

Unfortunately, two of our members did not follow this rule.

It would have been helpful as chair to have had an up-front, agreed-upon consequence to apply in this situation. Future CSS meetings certainly will have this in place. We should probably do this for TPAC, too.

**Comment by cwilso** [comment]
I definitely agree that consequences should be clearer (and enforced).  I'll note as an aside that the TPAC rule seems... less realistic than the CSSWG's, and would encourage rewording, as these do not appear to come from federal rules.  (Not eating or drinking in-flight, for those coming from the other side of the planet, seems unrealistic.).  Also, slight tweaking to clarify travel journey is relative to shared transit (I'm planning on driving my own vehicle to TPAC, and wasn't intending to mask up.  Is that a violation?); and the rules for travel should NOT be down in the "After your arrival" section.

**Comment by cwilso** [comment]
Also, where/what are the mask consequences?  I can't find any consequences.

**Comment by astearns** [comment]
“The access to the TPAC sessions may be refused if attendees do not agree to wear a mask.”

I think that could be strengthened to something like

“Access to TPAC will be revoked if attendees repeatedly fail to follow all the masking rules”

**Comment by dwsinger** [comment]
An important presumption behind both the CSS meeting and the upcoming TPAC was that if we agreed rules, that we were a cooperating, good faith, community and we could trust and assume that the rules would be followed and enforcement and policing were not needed.

This incident tells us that this presumption is, at best, seriously in question. TPAC is what, 10x or more the size of CSS.

**Comment by cwilso** [comment]
Ah!  I missed that line, thanks @astearns .  I agree with strengthening that wording.

**Comment by dwsinger** [comment]
It can only be strengthened if we have the willingness and ability to act. Can we rely on the 'chairs' to make rulings and the team to enforce them? We decided on trust because we didn't want to build this enforcement.

**Comment by stephenmcgruer** [comment]
> Not eating or drinking in-flight, for those coming from the other side of the planet, seems unrealistic.

Fwiw, the sentence "Wear a mask during your entire travel journey (onboard a flight, train or federally-regulated vessel, as well as in airports and other points of entry" is inside a section that is marked as "There are some federal requirements [...]". As such I assume it is attempting to summarize Canada's current federal requirements, but is doing so incorrectly, as far as I know (I am not a lawyer!). Federal rules allow the removal of a mask in both airports and onboard airplanes [in order to eat, drink, or take oral medication](https://tc.canada.ca/en/initiatives/covid-19-measures-updates-guidance-issued-transport-canada/covid-19-information-travellers-within-canada).

I believe only the text underneath "On-site W3C Rules" are TPAC-specific rules, as opposed to Canadian law requirements?

**Comment by frivoal** [comment]
The rules state:
> The access to the TPAC sessions may be refused if attendees do not agree to wear a mask.

Having “may“ in that sentence is a problem, as that invites debate between the participant and the person enforcing the rules. “will” would be much preferable.

Also, this appears to be the only rule for which consequences for non compliance are stated. All rules must have consequences, and these consequences must be described in advance in order not to invite controversy. 

If there are no consequences to non compliance, these are not rules, but mere guidelines (which could be useful too, but ought to be clearly separated from the rules).

**Comment by dwsinger** [comment]
I think that we should divide the 'rules' into two groups:

We trust that you, as a cooperating member of the community, will:
* a
* b

(These are unverifiable and unenforcable)

You must do:
* c
* d

and if you don't, you will be denied admission. (These are enforcable).

And finally, 

Please read the Canadian rules that apply to your travel and visit at reference-inserted. We do not summarize or interpret them here.

**Comment by frivoal** [comment]
I agree with @dwsinger 's suggestion above, and it is very similar to what I wrote in #12.
